src/styles/playlists.css

```diff
@@ -0,0 +1,25 @@
+.relative {
+    position: relative;
+}
+
+.no-playlist{
+    color: #a0b2b9;
```

Comment:
I see that this color value is being used twice (line no 6 and 16) in this file so use the color variable

Reply:
Okay.